docs/en/observability/uptime-set-up-app.asciidoc

```diff
@@ -0,0 +1,165 @@
+[[uptime-set-up-choose-agent]]
+= Use the {uptime-app}
+
+beta[]
+
+You can now create monitors directly in the {uptime-app}.
+To try this beta functionality, first you need to enable Monitor Management:
+
+. Go to **{observability}** -> **Uptime**.
+. Click **Monitor Management**.
+. Review the terms and conditions and click **Enable** to start using Monitor Management.
+
+From here, you can either <<global-managed-testing-infrastructure,use Elastic's global managed testing infrastructure>>
+or <<private-locations,create a private location>> to run monitors from your own premises.
+
+NOTE: To use the Elastic Synthetics integration, see the https://www.elastic.co/guide/en/observability/8.3/uptime-set-up.html#uptime-set-up-choose-agent[8.3 documentation]. The Elastic Synthetics integration is similar to the {uptime-app} approach, but contains fewer configuration options.
```

Comment:
There was some discussion about _fewer configuration options_ and some more content added in https://observability-docs_2075.docs-preview.app.elstc.co/guide/en/observability/master/uptime-set-up.html, maybe this should match?

Reply:
I'm not sure what this means. 🙃

Reply:
Sorry @colleenmcginnis, my main point was it should probably have the same content as https://observability-docs_2075.docs-preview.app.elstc.co/guide/en/observability/master/uptime-set-up.html

>To use the Elastic Synthetics integration, see the [8.3 documentation](https://observability-docs_2075.docs-preview.app.elstc.co/guide/en/observability/8.3/uptime-set-up.html#uptime-set-up-choose-agent). The Elastic Synthetics integrations is similar to the Uptime app approach described above, but contains fewer configuration options and does not allow you to run monitors against Elastic’s global managed testing locations.

I _thought_ there has been some discussion about the term _...fewer configuration options..._ but can't find that now, so apologies for the confusion.

Reply:
Oh I see! Thank you.